openeduhub/.htaccess

```diff
@@ -1,3 +1,16 @@
 RewriteEngine on
 RewriteRule ^vocabs/(.*) https://vocabs.openeduhub.de/$1 [R=302,L]
+RewriteRule ^vocabs/category/(.*) https://vocabs.openeduhub.de/w3id.org/openeduhub/vocabs/category/$1 [R=302,L]
```

Comment:
Rules are checked in order so the first "^vocabs/(.*)" will match before these new rules.  Probably just need to move the new ones so they are checked first.

Reply:
I'm sorry, I did not know that.
I corrected as suggested and will check beforehand next time. Thank you!